pages/dos/chdir.md

```diff
@@ -0,0 +1,7 @@
+# CHDIR
+
+> This command is an alias of `CD`.
+
+- View documentation for the original command:
+
+`tldr cd`
```

Comment:
I don't think anyone is going to be running a native tldr DOS client. Until such a thing pops up, I think it's reasonable to add --platform
```suggestion
`tldr cd {{[-p|--platform]}} dos`
```

Reply:
Seems reasonable. But I'd rather swap the command and the option. What do you think?
```suggestion
`tldr {{[-p|--platform]}} dos cd`
```

Reply:
We recently had a style guide change to put options at the end for clarity #18491
I'm open for different opinions and justifications for why the order should be different.

Reply:
OK, I see. But, personally, I don't like that change. We won't get a consistency for all commands, because some  require flags and options be placed before the positional arguments (ex. docker). Also, most of the documentation uses the options-arguments order, so most people are used to this order too (we're not just for newbies, right?), so this new order is very confusing for them. And now we need to change almost all the pages and translations. For what reason? IMO, regarding all this, this huge amount of work doesn't worth it.

Reply:
>We won't get a consistency for all commands

That's ok. Command functionality triumphs any style guide suggestion

>so this new order is very confusing for them

I don't think it is, but I would like to see an actual newbie tell their opinion

>And now we need to change almost all the pages and translations.

There's no *need* to do this. This is an incremental project and things get done when someone wants to do it.

I'm still actively thinking about these changes so we might change things.

Reply:
I've decided to go with the more common style in the ordering.

Also, I think that change in the style guide should be revised, as I noticed there wasn't much discussion about it, and there are some reasons against it.

Reply:
I think the current style guide is good thanks to this exception

> This is only a suggestion and should be disregarded when program functionality or readability dictates otherwise.

We'll just go case-by-case basis on what looks good.